pbnh/db/createdb.py

```diff
@@ -3,12 +3,14 @@
 from sqlalchemy_utils import create_database
 
 from pbnh import conf
+from pbnh import app
 from pbnh.db import models
 from pbnh.db.connect import DBConnect
 
+
 class CreateDB():
     def __init__(self, dialect=None, driver=None, username=None, password=None,
-                 host=None, port=None, dbname=None):
+                 host=None, port=None, dbname=app.app.config['DATABASE']):
```

Comment:
This isn't where you set the default values. You want `self.dbname = dbname or app.app.config['DATABASE']` I think. 

I was running into issues setting default values for this function specifically, which is why I did them below.